### PR TITLE
fix minor issues with initial install

### PR DESCRIPTION
**Pull request by bmenasha**

### CODE REVIEW

File: 2_gitlab/create-repos.sh [hunk 1]
```diff
@@ -29,6 +29,7 @@ pushd gitlab-repos
     for cluster in ${CLUSTERS}; do
        test -f ${cluster} || ssh-keygen -f ${cluster} -N ''
        # Create a secret with the private key
+       gcloud secrets delete gitlab-cluster-key-${cluster} --quiet || true
```
Comment (morgante) [inline 2_gitlab/create-repos.sh:32]: We should probably only do this in destroy, we don't want to delete these if someone already has them (but didn't destroy).

Comment (mike-ensor) [reply 2_gitlab/create-repos.sh:32]: new keys are created every time (line 30) so we need to delete the existing key and replace with a new.  Perhaps we can do versions instead, but the value does need to be replaced

Comment (morgante) [reply 2_gitlab/create-repos.sh:32]: Every time you rebuild the infra? Wouldn't destroy take care of that though?

If you think this is the right approach, that's fine though.

Comment (bmenasha) [reply 2_gitlab/create-repos.sh:32]: they are recreated in the very next line.. the script regenerates the keys and there is no api to update a secret version, only delete and recreate. 
thanks

Comment (mike-ensor) [reply 2_gitlab/create-repos.sh:32]: I'd like to keep this as-is for now.  Destroy does contain a removal of the secret. This is mostly used as a protection to ensure the key is stored in the event that the secret exists (providing idempotency)

**Comment by bmenasha** [comment]
thanks. changed to not log the password.

**Pull request merged**